pkg/downward/transform.go

```diff
@@ -94,17 +94,34 @@ func TransformConfigTemplatesWithApi(bootstrapConfig *envoy_config_bootstrap.Boo
 		return err
 	}
 
-	transformStruct(interpolate, bootstrapConfig.Node.Metadata)
+	err = transformStruct(interpolate, bootstrapConfig.Node.Metadata)
+	if err != nil {
+		return err
+	}
 
+	// Interpolate Static Resources
+	for _, cluster := range bootstrapConfig.GetStaticResources().GetClusters() {
+		for _, endpoint := range cluster.GetLoadAssignment().GetEndpoints() {
+			for _, lbEndpoint := range endpoint.GetLbEndpoints() {
+				socketAddress := lbEndpoint.GetEndpoint().GetAddress().GetSocketAddress()
+				if socketAddress != nil {
+					if err = interpolate(&socketAddress.Address); err != nil {
+						return err
+					}
+				}
+			}
+		}
+	}
```

Comment:
Is there some magic way I can convert `bootstrapConfig.GetStaticResources()` to a `structpb.Struct` in order to use `transformStruct(interpolate, bootstrapConfig.Node.Metadata)`?

There's inevitably going to be more fields in `StaticResources` that users will want to use interpolation on in the future. It'd be nice to just knock them all out at once.